src/api/index.ts

```diff
@@ -6,49 +6,33 @@
  */
 
 import axios from 'axios'
+import {stringify} from 'query-string'
 
-let api: any
 let baseUrl: string = process.env.API_ENDPOINT
 	? process.env.API_ENDPOINT
 	: 'https://jsonplaceholder.typicode.com'
```

Comment:
Could be shortened to `: string = process.env.API_ENDPOINT || 'https://jsonplaceholder.typicode.com'`

Reply:
`baseUrl` should be const as well